README.md

```diff
@@ -37,17 +35,63 @@ The logging-ocm-addon consists of one component:
 $ kubectl apply -k deploy/
 ```
 
-2. The addon should now be installed in you hub cluster 
+1. The addon should now be installed in you hub cluster
+
 ```shell
 $ kubectl get ClusterManagementAddOn multicluster-observability-addon
 ```
 
-3. The addon can now be installed it managed clusters by creating `ManagedClusterAddOn` resources in their respective namespaces
+1. The addon will install automatically in spoke clusters once the resources referenced in `ClusterManagementAddOn` are created.
+
+#### Installing via MCO
+
+In 2.12, multicluster-observability-operator has the ability to install MCOA using the [capabilities field](https://github.com/stolostron/multicluster-observability-operator/blob/5d1fc789df365b20951b5fe1c378b5eebb306390/operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go#L187-L212).
+
+1. Create a `MultiClusterObservability` resource and configure `capabilities`
+
+```yaml
+apiVersion: observability.open-cluster-management.io/v1beta2
+kind: MultiClusterObservability
+metadata:
+  name: observability
+spec:
+  capabilities:
+    platform:
+      logs:
+        collection:
+          enabled: true
+    userWorkloads:
+      logs:
+        collection:
+          clusterLogForwarder:
+            enabled: true
+      traces:
+        collection:
+          instrumentation:
+            enabled: true
+          openTelemetryCollector:
+            enabled: false
+  observabilityAddonSpec: {}
+  storageConfig:
+    metricObjectStorage:
+      name: thanos-object-storage
+      key: thanos.yaml
+```
+
+Note: Deploy a custom image by adding the annotation: `mco-multicluster_observability_addon-image: quay.io/YOUR_ORG_HERE/multicluster-observability-addon:YOUR_TAG_HERE`
```

Comment:
The "counting enumeration" only continues when you indent this with 4 spaces:

```suggestion
    ```yaml
    apiVersion: observability.open-cluster-management.io/v1beta2
    kind: MultiClusterObservability
    metadata:
      name: observability
    spec:
      capabilities:
        platform:
          logs:
            collection:
              enabled: true
        userWorkloads:
          logs:
            collection:
              clusterLogForwarder:
                enabled: true
          traces:
            collection:
              instrumentation:
                enabled: true
              openTelemetryCollector:
                enabled: false
      observabilityAddonSpec: {}
      storageConfig:
        metricObjectStorage:
          name: thanos-object-storage
          key: thanos.yaml
    ```

    Note: Deploy a custom image by adding the annotation: `mco-multicluster_observability_addon-image: quay.io/YOUR_ORG_HERE/multicluster-observability-addon:YOUR_TAG_HERE`
```